samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ConfirmPublishAssessmentListener.java

```diff
@@ -183,11 +183,11 @@ public void processAction(ActionEvent ae) throws AbortProcessingException {
 		  Date entryStartDate = entry.getStartDate();
 		  Date entryDueDate = entry.getDueDate();
 		  Date entryRetractDate = entry.getRetractDate();
-		  if(!"".equals(entry.getUser())) {
+		  if(!StringUtils.isEmpty(entry.getUser())) {
```

Comment:
isNotEmpty is preferred over !isEmpty

Reply:
Hi @ottenhoff
I've made the change. Regards